### PR TITLE
How to hide a user from a TalkJS Chat

### PR DESCRIPTION
**Pull request by aswinrajeevofficial**
This PR contains the code to go with the tutorial on How to add a GIF picker to your TalkJS chat.

Article: https://docs.google.com/document/d/1XjpeepDguZeZs7aW2rkubiNIWzGNdr2LOxbQmspD8Bc/edit?usp=sharing

### CODE REVIEW

File: howtos/how-to-add-a-gif-picker-to-your-talkjs-chat/README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 This is an example project to go with our tutorial on How to add a GIF picker to your TalkJS chat.
-<!-- Add link to tutorial once published -->
+
+[How to add a GIF picker to your TalkJS chat](https://talkjs.com/resources/how-to-add-a-gif-picker-to-your-talkjs-chat/)
```
Comment (keerlu) [inline howtos/how-to-add-a-gif-picker-to-your-talkjs-chat/README.md:3]: Good catch 😁

File: howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+This is an example project to go with our tutorial on how to add a hidden user to your TalkJS chat.
```
Comment (keerlu) [inline howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md:1]: Reminder to myself to add a link after publishing

File: howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+<!-- Add link to tutorial once published -->
+
+This project adds a new [TalkJS Theme](https://talkjs.com/docs/Features/Themes/) and edits different components within it to hide a user that has a custom property.
+
+## Prerequisites
+
+To run this tutorial, you will need:
+
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+-
```
Comment (keerlu) [inline howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md:12]:
```suggestion

```

File: howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+## Prerequisites
+
+To run this tutorial, you will need:
+
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+- 
+
+## How to run the tutorial
+
+1. Clone or download this project.
```
Comment (keerlu) [inline howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md:16]: Need one more line on adding the App ID

File: howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+2. Go to your TalkJS dashboard and navigate to the **Themes** tab. Create a new theme called "hidden_user" based on the "default" theme.
+3. Add a custom attribute and the "default" role to all your users in the group chat.
```
Comment (keerlu) [inline howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md:18]: I'd suggest swapping these two steps to have all the theme-related stuff together

```suggestion
2. Add a custom attribute and the "default" role to all your users in the group chat.
3. Go to your TalkJS dashboard and navigate to the **Themes** tab. Create a new theme called "hidden_user" based on the "default" theme.
```

**Review by keerlu — APPROVED** [review]
Looks good! Some minor comments

File: howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/index.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
+let conversation;
+
+Talk.ready.then(() => {
+  const me = new Talk.User({
```
Comment (keerlu) [inline howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/index.js:4]: It was slightly confusing to me to see the inbox from the point of view of the hidden user, I was expecting to see it from the point of view of a normal user, so it was clear that one participant was missing. So I'd suggest swapping them round

Comment (aswinrajeevofficial) [reply howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/index.js:4]: Got it

File: howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md (hunk 5 above)
Comment (aswinrajeevofficial) [reply howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md:18]: I think adding the custom attribute and default role to all users in the group chat is not needed if they're cloning this project.

**Comment by aswinrajeevofficial** [comment]
I've made the changes to the article and the PR as per your feedback @keerlu. Let me know what you think.

File: howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/index.js (hunk 6 above)
Comment (keerlu) [reply howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/index.js:4]: Thanks, that does look clearer to me!

File: howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md (hunk 2 above)
Comment (keerlu) [reply howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md:1]: (looks like we're going to merge code first, so making a reminder elsewhere)

**Pull request merged**